mmcv/runner/hooks/logger/wandb.py

```diff
@@ -14,13 +18,28 @@ def __init__(self,
                  reset_flag=False,
                  commit=True,
                  by_epoch=True,
-                 with_step=True):
+                 with_step=True,
+                 config_path=None):
+        
         super(WandbLoggerHook, self).__init__(interval, ignore_last,
                                               reset_flag, by_epoch)
+        
+        """
+        Args:
+            with_step (bool): whether making a log in each step or not.
+                Default: True.
+            config_path (str, optional): The path of the final config of each
+                experiment. It can be either a path of the final config file or
+                a directory of config files. The config is uploaded to wandb
+                server if it is not None. Default: None.
+                `New in version 1.3.18.` 
+        """
```

Comment:
suggest moving those lines to line13.

Reply:
You are right. It is moved.

Reply:
@zhouzaida After moving it, docs reader make build fail. Should I indicate the docs is for `__init__`? Please let me know about it.

Reply:
@zhouzaida Sorry, it was version issue, my mistake. I didn't update my forked repo so long time...